internal/nostr/nostr.go

```diff
@@ -948,6 +1006,9 @@ func (svc *Service) subscriptionToFilter(subscription *Subscription) (*nostr.Fil
 }
 
 func (svc *Service) getWalletPubkey(authors *[]string) string {
+	fmt.Println("authors")
+	fmt.Println(authors)
+	fmt.Println("authors")
```

Comment:
you should never commit all.
when you commit changes ALWAYS look at the diff and commit selectively. 
never use the commit from your editor. use the git CLI or some git client and check what you commit. 

You too often commit such debug lines.

Reply:
Got it 😬